docs/v2-system-addon/data_events.md

```diff
@@ -195,3 +195,81 @@ Here are different scenarios that cause a session end event to be sent:
 3. Closing the browser
 5. Refreshing
 6. Navigating to a new URL via the url bar or file menu
+
+### Session performance data
+
+This data is held in a child object of the ```activity_stream_session``` event called ```perf```.  All fields suffixed by ```_ts``` are type ```DOMHighResTimeStamp``` (aka a double of milliseconds, with a .5 microsecond precision) with 0 being the timeOrigin of the browser's hidden
+chrome window.
```

Comment:
Is there maybe a slightly clearer way to describe "the timeOrigin of the browser's hidden chrome window"? Is timeOrigin (`timeOrigin`?) a specific thing in this context? If so would a link/small description help?